docs/Tutorials/Create-Private-Clique-Network.md

```diff
@@ -0,0 +1,271 @@
+
+In the `Node-1` directory, use the [`export-pub-key`](../Reference/Pantheon-CLI-Syntax.md#export-pub-key) subcommand to write 
+the [node public key](../Configuring-Pantheon/Node-Keys.md) to the specified file (`publicKeyNode1` in this example):
+
+```bash tab="MacOS"
+pantheon --data-path=Node-1-data-path public-key export --to=Node-1-data-path/publicKeyNode1
+```
+
+```bash tab="Windows"
+pantheon --data-path=Node-1-data-path public-key export --to=Node-1-data-path\publicKeyNode1
+```
+
+Your node 1 directory now contains: 
+```bash
+├── Node-1
+    ├── Node-1-data-path
+        ├── database
+        ├── key
+        ├── publicKeyNode1
+```
+      
+The `database` directory contains the blockchain data. 
+
+### 3. Get Address for Node-1 
+
+In Clique networks, the address of at least one initial signer must be included in the genesis file. 
+For this Clique network, we will use Node-1 as the initial signer. This requires obtaining the address for Node-1. 
+
+To obtain the address for Node-1, do one of the following with the private key in the `key` file in the `Node-1-data-path` directory: 
+
+* Import the private key into [MetaMask](https://metamask.io/) and click the **Copy to clipboard** button 
+displayed when hovering over the account name and address. 
+* Use the [ethereumjs-util](https://github.com/ethereumjs/ethereumjs-util) library and [node](https://nodejs.org/en/) 
+to execute the following where `<private key>` is replaced by the private key for Node-1 without the 0x prefix. 
+```js
+var ethUtil = require('ethereumjs-util')
+const privKey = Buffer.from('<private key>', 'hex')
+var address = ethUtil.privateToAddress(privKey).toString('hex')
+console.log("Address=" + address)
+```
+
+### 4. Create Genesis File 
+
+The genesis file defines the genesis block of the blockchain (that is, the start of the blockchain).
+The [Clique genesis file](../Consensus-Protocols/Clique.md#genesis-file) includes the address of Node-1 as the initial signer in the `extraData` field.    
+
+All nodes in a network must use the same genesis file. 
+
+Copy the following genesis definition to a file called `cliqueGenesis.json` and save it in the `Clique-Network` directory: 
+
+```json
+{
+  "config":{
+    "chainId":1981,
+    "clique":{
+      "blockperiodseconds":15,
+      "epochlength":30000
+    }
+  },
+  "coinbase":"0x0000000000000000000000000000000000000000",
+  "difficulty":"0x1",
+
+"extraData":"0x0000000000000000000000000000000000000000000000000000000000000000<Node 1 Address>0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+  "gasLimit":"0xa00000",
+  "mixHash":"0x0000000000000000000000000000000000000000000000000000000000000000",
+  "nonce":"0x0",
+  "timestamp":"0x5c51a607",
+  "alloc": {
+    "<Node 1 Address>": {
+      "privateKey": "<Node 1 Private Key>",
```

Comment:
are we sure the private key is necessary in the Genesis file ?

Reply:
I did go back and forth about this.  By including it they'll have an account with a balance they can use for transactions. Maybe I'll rethink it.

Reply:
I'm removing this and including the test accounts from dev.json because we already use them in the other tutorials so they'll have accounts with balances to use on this network. I'll include Adrian's point.